sc_rf_service/sc_rf_service.py

```diff
@@ -1,496 +1,389 @@
+                       enum_strings=("Ok", "Fault"))
+    rackB = pvproperty(value=0, name=":CPLRVACB_LTCH", dtype=ChannelType.ENUM,
+                       enum_strings=("Ok", "Fault"))
+
+
+class StepperPVGroup(PVGroup):
+    move_pos = pvproperty(name="MOV_REQ_POS")
+    move_neg = pvproperty(name="MOV_REQ_NEG")
+    abort = pvproperty(name="ABORT_REQ")
+    step_des = pvproperty(name="NSTEPS")
+    max_steps = pvproperty(name="NSTEPS.DRVH")
+    speed = pvproperty(name="VELO")
+    step_tot = pvproperty(name="REG_TOTABS")
+    step_signed = pvproperty(name="REG_TOTSGN")
+    reset_tot = pvproperty(name="TOTABS_RESET")
+    reset_signed = pvproperty(name="TOTSGN_RESET")
+    steps_cold_landing = pvproperty(name="NSTEPS_COLD")
+    push_signed_cold = pvproperty(name="PUSH_NSTEPS_COLD.PROC")
+    push_signed_park = pvproperty(name="PUSH_NSTEPS_PARK.PROC")
+    motor_moving = pvproperty(name="STAT_MOV")
+    motor_done = pvproperty(name="STAT_DONE")
+
+
+class PiezoPVGroup(PVGroup):
+    enable = pvproperty(name="ENABLE")
+    feedback_mode = pvproperty(value=1, name="MODECTRL",
+                               dtype=ChannelType.ENUM,
+                               enum_strings=("Manual", "Feedback"))
+    dc_setpoint = pvproperty(name="DAC_SP")
+    bias_voltage = pvproperty(name="BIAS")
+    prerf_test_start = pvproperty(name="TESTSTRT")
+    prerf_cha_status = pvproperty(name="CHA_TESTSTAT")
+    prerf_chb_status = pvproperty(name="CHB_TESTSTAT")
+    prerf_cha_testmsg = pvproperty(name="CHA_TESTMSG1")
+    prerf_chb_testmsg = pvproperty(name="CHA_TESTMSG2")
+    capacitance_a = pvproperty(name="CHA_C")
+    capacitance_b = pvproperty(name="CHB_C")
+    prerf_test_status = pvproperty(name="TESTSTS")
+    
+    withrf_run_check = pvproperty(name="RFTESTSTRT")
+    withrf_check_status = pvproperty(name="RFTESTSTS")
+    withrf_status = pvproperty(name="RFSTESTSTAT")
+    amplifiergain_a = pvproperty(name="CHA_AMPGAIN")
+    amplifiergain_b = pvproperty(name="CHB_AMPGAIN")
+    withrf_push_dfgain = pvproperty(name="PUSH_DFGAIN.PROC")
+    withrf_save_dfgain = pvproperty(name="SAVE_DFGAIN.PROC")
+    detunegain_new = pvproperty(name="DFGAIN_NEW")
+    hardware_sum = pvproperty(value=0, name="HWSTATSUM", dtype=ChannelType.ENUM,
+                              enum_strings=("", "", "Fault"))
+    feedback_sum = pvproperty(value=0, name="FBSTATSUM",
+                              dtype=ChannelType.ENUM,
+                              enum_strings=("", "", "Fault"))
+
+
+class CavFaultPVGroup(PVGroup):
+    cryoSummary: PvpropertyEnum = pvproperty(value=0, name="CRYO_LTCH",
+                                             dtype=ChannelType.ENUM,
+                                             enum_strings=("Ok", "Fault"))
+    resLinkLatch: PvpropertyEnum = pvproperty(value=0, name="RESLINK_LTCH",
+                                              dtype=ChannelType.ENUM,
+                                              enum_strings=("OK", "Fault"))
```

Comment:
I notice the names no longer have a ":" in them. Does this get accounted for somewhere else now? Aka it used to be `name=":CRYO_LTCH"` but now it's just `name="CRYO_LTCH"`

Reply:
Yup,I added the colon to the prefix! That made more sense to me